yarn.lock

```diff
@@ -508,6 +508,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/parser@npm:^7.24.4":
```

Comment:
I think this shall not be here. Maybe `yarn dedupe`?

Also we shall add this to pipeline as well (if dedupe depdupes something -> fail)

Reply:
fixed in commit [3161f27](https://github.com/trezor/trezor-suite/pull/12914/commits/3161f274b321cf362c0e2defad9c4945547e8943)

good call :) 
![Screenshot 2024-06-18 at 7 38 40](https://github.com/trezor/trezor-suite/assets/26143964/d02ef472-1e2b-4ff3-9c4d-06933496998d)